index.html

```diff
@@ -57,13 +64,181 @@ <h1>TechConnect <br> Web Development Workshop</h1>
+                </div>
+
+            </div>
+
+            <div class="card">
+                <i class="fa-duotone fa-book"></i>
+                <h5>Web Development</h5>
+                <div class="pra">
+                     <p>Overview of web technologies, front-end, and back-end development</p>
+                </div>
+
+            </div>
 
+        </div>
+
+        <p class="see"><a href="#">SEE THE WHOLE PROGRAM</a></p>
         
-    </section> -->
+          
+    </section>
+
+    <section class="speakers">
+      <div class="title2">
+        <h2>Featured Speakers 
+            <hr class="section-title_line">
+        </h2>
+      </div>
+      <div class="speakers2">
+
+          <div class="main">
+            <img src="./speaker.jpg">
+            <div class="about-text">
+                <h2>Dorcas Amini</h2>
+                <h5>profeseur en IT</h5>
+                <P>Dorcas is a seasoned web developer with over 8 years of experience. She has worked on a wide range.</P>
+    
+            </div>
+    
+          </div>
+    
+          <div class="main">
+            <img src="./speaker.jpg">
+            <div class="about-text">
+                <h2>Dorcas Amini</h2>
+                <h5>profeseur en IT</h5>
+                <P>>Dorcas is a seasoned web developer with over 8 years of experience. She has worked on a wide range</P>
+    
+            </div>
+    
+          </div>
+
+          <div class="main">
+            <img src="./speaker.jpg">
+            <div class="about-text">
+                <h2>Dorcas Amini</h2>
+                <h5>profeseur en IT</h5>
+                <P>>Dorcas is a seasoned web developer with over 8 years of experience. She has worked on a wide range</P>
+    
+            </div>
+    
+          </div>
+          <div class="main">
+            <img src="./speaker.jpg">
+            <div class="about-text">
+                <h2>Dorcas Amini</h2>
+                <h5>profeseur en IT</h5>
+                <P>>Dorcas is a seasoned web developer with over 8 years of experience. She has worked on a wide range</P>
+    
+            </div>
+    
+          </div>
+    
+          <div class="main">
+            <img src="./speaker.jpg">
+            <div class="about-text">
+                <h2>Dorcas Amini</h2>
+                <h5>profeseur en IT</h5>
+                <P>>Dorcas is a seasoned web developer with over 8 years of experience. She has worked on a wide range</P>
+    
+            </div>
+    
+          </div>
+    
+          <div class="main">
+            <img src="./speaker.jpg">
+            <div class="about-text">
+                <h2>Dorcas Amini</h2>
+                <h5>profeseur en IT</h5>
+                <P>>Dorcas is a seasoned web developer with over 8 years of experience. She has worked on a wide range</P>
+    
+            </div>
+    
+          </div>
+           
+    
+           
+
```

Comment:
- [ ] `Required`
- [ ] The "Featured speakers" segment is dynamically generated using JavaScript. The JavaScript code resides in a separate .js file rather than being embedded within the HTML using script tags. The collection of speakers, which is employed to generate the HTML, is stored as an array or object within a variable. Each speaker within the list is represented by a speaker object that encapsulates their distinct data.
Please make amends 🥂